### ISSUE TITLE
Termux - error when executing DAG in web UI due to missing /tmp in Termux

### ISSUE THREAD

**Issue by edwindvinas** [yohamta/dagu#299]
**Minimal reproducible example DAG**
Installed dag binary inside Termux in Android. I was able to run the dagu server on port 6060. I can access the UI via browser and created the example DAG as specified in the README. But when I try to execute the DAG, I can see that its due to no access to /tmp.

It can't find the /tmp since Termux does not have access to /tmp due to Permission denied.
It only has access from /data/data/com.termux/files/home

```
Log File: 

/data/data/com.termux/files/home/.dagu/logs/dags/example/agent_example.20220829.16:56:56.522.c852ff56.log
```

```
2022/08/29 16:56:56 failed to start socket server listen unix /tmp/@dagu-example-d7b419054d0b13db8f4aa83ac3905be7.sock: bind: no such file or directory
```

**Expected behavior**
If dagu server can verify if it has access to /tmp otherwise use a different path.
Not sure about the error though as I only tried it via Termux.
https://termux.dev/en/

Basically, the issue is that dagu can't access /tmp/@dagu-example-d7b419054d0b13db8f4aa83ac3905be7.sock in my Termux environment.

**Comment by edwindvinas** [comment]
Apologies, this is no longer an issue since there is a workaround. I can now run Dagu in Termux.
https://www.reddit.com/r/termux/comments/hnqbd4/cant_modify_readonly_file_system/
